apps/server/src/services/project-service/ProjectService.ts

```
@@ -93,6 +93,14 @@ async function loadProject(projectData: DatabaseModel, fileName: string) {
 
   // load the first rundown in the project
   const firstRundown = getFirstRundown(projectData.rundowns);
+
+  // eslint-disable-next-line no-unused-labels -- dev code path
+  DEV: {
```

Biome (1.9.4)

[error] Unexpected label.

Only loops should be labeled.
The use of labels for other statements is suspicious and unfamiliar.

(lint/suspicious/noConfusingLabels)